packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -147,11 +147,10 @@ class TransferEngine implements ITransferEngine {
 
       // Gracefully close the providers
       await this.close();
-    } catch (e) {
-      console.log('error', e);
+    } catch (e: unknown) {
       // Rollback the destination provider if an exception is thrown during the transfer
       // Note: This will be configurable in the future
-      // await this.destinationProvider?.rollback(e);
+      await this.destinationProvider.rollback?.(e as Error);
```

Comment:
Just wondering if there's any additional context rollback could find useful besides the error. But I can't actually think of anything, so this is probably fine.

Reply:
You can leave it, better to delete the file on error.